Install.md

```diff
@@ -3,79 +3,61 @@
 This guide helps you build and install Snort.
 
 ---
-1. Compile DPDK
+1. Setup Repositories
 ---
 
-1. Export correct target variable for your system as required by DPDK.
-
+1. Download source code
     ```sh
-    export RTE_TARGET=x86_64-native-linuxapp-gcc
-    export DPDK_TARGET=x86_64-native-linuxapp-gcc
+    git clone https://github.com/sdnfv/onvm-snort
+    cd onvm-snort
     ```
 
-2.  Navigate to your DPDK source repository and set environment variable RTE_SDK to the path of the DPDK library. 
+2. Initialize openNetVM submodule
     ```sh
-    cd dpdk-stable-16.11.1
-    export RTE_SDK=$(pwd)
+    git submodule sync
+    git submodule update --init
     ```
-
-3. Build DPDK.
-    ```sh
-    make config T=x86_64-native-linuxapp-gcc O=x86_64-native-linuxapp-gcc
-    make -j7 T=$RTE_TARGET O=$RTE_TARGET
-    sudo make install T=x86_64-native-linuxapp-gcc
-    ```  
+    
 ---
-2. Compile ONVM
+2. Compile DPDK and openNetVM
 ---
 
-1. Navigate to openNetVM source directory.
-    ```sh
-    cd openNetVM/
-    export ONVM_HOME=$(pwd)
-    ```
-    
-2. Compile onvm.
-    ```sh
-    cd onvm && make
-    ```
+Please follow [the openNetVM installation guide](https://github.com/sdnfv/openNetVM/blob/master/docs/Install.md).
+
 ---
 3. Compile DAQ
 ---
 
 1. Install dependencies.
     ```sh
-    sudo apt-get install -y libpcap-dev libpcre3-dev libdumbnet-dev zlib1g-dev liblzma-dev libssl-dev autoconf
+    sudo apt-get install -y libpcap-dev libpcre3-dev libdumbnet-dev zlib1g-dev liblzma-dev libssl-dev autoconf flex bison luajit libtool libglib2.0-dev pkg-config
     ```
 
 2. Navigate to the DAQ source directory.
     ```sh
     cd daq-2.0.6/
     ```
-
-2. Clean the build. 
-    ```sh
-    make clean
-    ```
     
 3. Prepare for automake, then autocreate makefile. 
     ```sh
     aclocal
     autoconf
     autoheader
+    autoreconf -fvi
```

Comment:
this should be `autoreconf -ivf` the current order causes an error in running `make -j7 ` in which tokdef.h is not found